live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -233,3 +233,41 @@
   with_items:
     - { key: "net.ipv4.conf.all.promote_secondaries", value: "1" }
     - { key: "net.ipv4.conf.default.promote_secondaries", value: "1" }
+
+#
+# Configure command audit logging
+#
+# We want to record all commands executed on the appliance. Opt out for
+# setsid since all ExecuteUtils.execute wrap each call with setsid.
+#
+# Note we can't use the "service" module, as that fails when running
+# inside the chroot environment, so we enable auditd using "command".
+#
+- apt:
+    name: auditd
+    state: installed
+    update_cache: no
+
+- command: /bin/systemctl enable auditd.service
+
+- lineinfile:
+    dest: /etc/audit/auditd.conf
+    regexp: "{{item.regex}}"
+    line: "{{item.line}}"
+  with_items:
+    - { regex: "^num_logs =", line: "num_logs = 6" }
+    - { regex: "^max_log_file =", line: "max_log_file = 3072" }
+    - { regex: "^max_log_file_action =", line: "max_log_file_action = rotate" }
+    - { regex: "^log_format =", line: "log_format = RAW" }
+
+- lineinfile:
+    dest: /etc/audit/rules.d/audit.rules
+    line: "{{item}}"
+    insertafter: EOF
+    state: present
+  with_items:
+    - "## Record all executed commands"
+    - "-a exit,never -S execve -F exe=/usr/bin/setsid"
+    - "-a exit,always -S execve"
+    - "-a exit,always -F a0!=0 -S exit_group"
```

Comment:
Maybe add a comment here explaining that this enables the tracking of the exit code of commands since the execve syscall will succeed if it's able to execute a command even if that command exits with a failure.